app/javascript/views/referrals/index.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react'
+
+import {
+  Wrapper,
+  Container,
+  Grid
+} from './style.ts'
+
+import ReferralsNav from './referralsNav/index.tsx'
+import ReferralsHeader from './referralsHeader/index.tsx'
+import ReferralsCard from './referralsCard/index.tsx'
+
+export default class Referrals extends React.Component {
+
+  render () {
+    return (
+      <Wrapper>
+        <ReferralsNav/>
+        <Container>
+          <ReferralsHeader/>
+          <Grid>
+            <ReferralsCard/>
```

Comment:
Can we name this component `ReferralCard` as opposed to `Referrals`? Each of these items is one Referral so I think that would make sense.

Reply:
Rethought this, is it okay if we keep as Referrals? Each card will be representing multiple codes, and I also like the consistency of `ReferralsX` components in the `Referrals` section